### DISCUSSION TITLE
You have no idea how much I use this extension

### DISCUSSION THREAD

**Discussion by GeekyGhost** [zero01101/openOutpaint#222] [Show and tell]
Thanks for making this.  

This is by far one of the best extensions made for Automatic1111. This and Controlnet both are crucial for me. 

![00109-964898542](https://user-images.githubusercontent.com/111990299/229569648-c2cfc366-766f-4503-9963-73454603f1ad.png)
![00150-1897297538](https://user-images.githubusercontent.com/111990299/229569690-84c3c684-694d-4244-91cb-bce9a3680367.png)

**Comment by zero01101** [comment]
aww <3 thanks muchly, glad people are still finding use for this silly little thing now that controlnet and modelscope are the new hotness lol :D

**Reply by seijihariki** [reply]
Hi, @zero01101!
Hey, really sorry for the long hiatus. Have been hung up with some work and playing a lot with langchain and other LLM stuff.

Was thinking of implementing something using meta's segmentation thing. What do you think of adding this to the extension?

[Edit]
I think it could be used to generate masks or create segmented layers of things. Imagine clicking on an object and it popping up to another layer, and using SD to automagically fill the gap left under the object.

**Reply by zero01101** [reply]
hey! welcome back! :D absolutely no worries at all of course; i've obviously been more absent than usual myself, life is bonkers everywhere lol :|  

i've barely even looked into (and much less TRIED) the segment-anything-doodad thingy yet; i'd _personally_ still like to keep openOutpaint vanilla without any external dependencies (well minus A1111 of course lmao) but i gotta say the few times i looked at the abstract it looked pretty impressive... 

>and other LLM stuff

lmao wtf is this small convergent world, i played a bit with llama/oobabooga and my m40 is apparently too old and my 2080ti isn't BIG ENOUGH to really play with 😵

**Reply by zero01101** [reply]
>Was thinking of implementing something using meta's segmentation thing. What do you think of adding this to the extension?  

https://github.com/continue-revolution/sd-webui-segment-anything/issues/15  👀 👀 

[edit] 

https://github.com/Mikubill/sd-webui-controlnet/issues/736#issuecomment-1506764538 👀 👀 👀

**Comment by LanceGao97** [comment]
I  think  your OpenOutPaint Almost have allthe  creating fucntions of StableDiffusion,  I would really wish Control Net and those Loras are in your OpenOutPaint rather than in webui,   webui is not for human, your product is like professional thing,  userfriendly and amazingly convinient,  You remind me of SteveJobs at 2010 year.
